status-head.js

```diff
@@ -1,31 +1,34 @@
+const Dynastar = require('dynastar');
+const Joi = require('joi');
+
 const Wrap = require('./wrap');
 
 /**
- * Returns a wrapped StatusHead model which is used for storing the latest
+ * Returns a wrapped Dynastar model which is used for storing the latest
  * status for a given pkg, env without needing to specify the version
  *
- * @function stathead
- * @param {Datastar} datastar Datastar instance
+ * @function statushead
+ * @param {Object} dynamo Dynamo object model
  * @returns {Wrap} StatusHead
  */
-module.exports = function stathead(datastar) {
-  const cql = datastar.schema.cql;
-  const StatusHead = datastar.define('status_head', {
-    schema: datastar.schema.object({
-      pkg: cql.text(),
-      env: cql.text(),
-      version: cql.text(),
-      previous_version: cql.text(),
-      total: cql.int(),
-      create_date: cql.timestamp({ default: 'create' }),
-      update_date: cql.timestamp({ default: 'update' })
-    }).partitionKey(['pkg', 'env']),
-    with: {
-      compaction: {
-        class: 'LeveledCompactionStrategy'
-      }
+module.exports = function statushead(dynamo) {
+  const hashKey = 'key';
+  const model = dynamo.define('status_head', {
```

Comment:
```suggestion
  const model = dynamo.define('StatusHead', {
```

lets change this snake_case to PascalCase.